keymage.js

```diff
@@ -8,8 +8,7 @@
 (function(define, undefined) {
 define(function() {
     var VERSION = '1.1.3';
-    var isOsx = typeof navigator !== 'undefined' &&
-        ~navigator.userAgent.indexOf('Mac OS X');
+    var isOsx = /^mac/i.test(navigator.platform);
```

Comment:
Furthermore, `Mac OS X` string appears to be in iOS UA strings: it is great cause any keyboard connected to iOS will use command/windows instead of control.

Brought it back. Landed couple optimizations. Thanks for reading all that ^^.